packages/blip/app/components/chart/daily.js

```diff
@@ -121,9 +123,8 @@ class DailyChart extends React.Component {
     if (chart !== null) {
       chart.loadingInProgress = loading
       if (needRecreate && !loading && !chart.isInTransition()) {
-        this.setState({ needRecreate: false }, () => {
-          this.reCreateChart()
-        })
+        this.setState({ needRecreate: false })
```

Comment:
This is because there is a breaking change since React 16 and Functionnal Components. With older class components the setState method can have a second argument that is a callback executed after the state has changed. But in React hooks the setState method no longer exists and the state setter takes only one argument (the state value). Now the main wrapper is written with a functionnal component, I don't know why but the setState callbacks where not called anymore (maybe the React magic sauce). Putting it just after the setState finally works !!